plugins/jobtype/src/azkaban/jobtype/HadoopSparkJob.java

```diff
@@ -348,9 +414,51 @@ private static void handleStandardArgument(Props jobProps,
       }
     }
 
+    info("Final classpath: " + classPath);
     return classPath;
   }
 
+  private String[] getSparkHomeConf() {
+    String sparkHome = null;
+    String sparkConf = null;
+    // If user has specified version in job property. e.g. spark.version=1.6.0
+    String jobSparkVer = getJobProps().get(SparkJobArg.SPARK_VERSION.azPropName);
+    if (jobSparkVer != null) {
+      // Spark jobtype supports this version through plugin's jobtype config
+      // e.g. spark.1.6.0.home=/path_to_spark/ in commonprivate.properties
+      sparkHome = getSysProps().get("spark." + jobSparkVer + ".home");
+      if (sparkHome != null) {
```

Comment:
This is a good suggestion. We should add log message to discriminate between no spark-version  and not-supported spark-version from the user